docs/install/install-cave.md

```diff
@@ -51,6 +51,12 @@ Regardless of what Operating System CAVE is running on, these general requiremen
 
 !!! note "While CentOS8 has reach End of Life as of Dec. 31, 2021, CentOS7 End of Life isn't until June 30, 2024."
 
+### Upgrade Existing Installation
```

Comment:
I feel like this is confusing...

"Whether you already have CAVE installed or not, you can follow the [Download and Installation Instructions](#download-and-installation-instructions) below. If installed the script will remove the old version and install the latest."